src/systemd-sonic-generator/systemd-sonic-generator.c

```diff
@@ -25,6 +25,7 @@ const char* CONFIG_FILE = "/etc/sonic/generated_services.conf";
 const char* MACHINE_CONF_FILE = "/host/machine.conf";
 const char* ASIC_CONF_FORMAT = "/usr/share/sonic/device/%s/asic.conf";
 const char* PLATFORM_FILE_FORMAT = "/usr/share/sonic/device/%s/platform.json";
+const char* PLATFORM_CONF_FORMAT = "/usr/share/sonic/device/%s/generated_services.conf";
```

Comment:
Is this file generated? I guess not. Let's change the name to something like `services.conf`.